### DISCUSSION TITLE
Icon request: fa-restaurant-table

### DISCUSSION THREAD

**Discussion by jaaksarv** [FortAwesome/Font-Awesome#21139] [Icon Requests]
I'm using Font Awesome for Restaurant Point-of-Sale System (http://speedpos.com). I missing an icon to use for restaurant tables.
Few ideas how it should look like: 
http://direct-chairs.com/images/ProductDetails/RestoTableRound.png
https://cdn2.iconfinder.com/data/icons/windows-8-metro-style/128/table.png
http://www.factorydirectlinen.com/images/ico-Table.jpg
http://www.secondhandpubfurniture.co.uk/images/chunky%20dining%20rectangle%20so%20-%20icon%20-%20medium.jpg
Choose the one that best fits Font Awesome style. Thank you:)

**Comment by jaaksarv** [comment]
Related issue for restaurant seats. It would be great if they play nicely together:
https://github.com/FortAwesome/Font-Awesome/issues/4150

**Comment by jaaksarv** [comment]
One old request that also included restaurant table:
https://github.com/FortAwesome/Font-Awesome/issues/3339

**Comment by mgntrn** [comment]
+1

**Comment by ianmartorell** [comment]
+1

**Comment by FrazzApps** [comment]
+1

**Comment by rok-safaric** [comment]
+1

**Comment by felipefrancisco** [comment]
+1

**Comment by rok-safaric** [comment]
+1

**Comment by kwaaui** [comment]
+1

**Comment by sarahg** [comment]
+1

**Comment by evinkuraga** [comment]
+1

**Comment by cnovarino** [comment]
+1

**Comment by pjfsilva** [comment]
+1

**Comment by blkdr** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by guylhermetabosa** [comment]
+1

**Comment by presercomp** [comment]
+1

**Comment by nairdo** [comment]
+1 if the table and chairs are separate but can be stacked to achieve this from the other example: https://cdn1.iconfinder.com/data/icons/kitchen-furniture-and-accessories/256/table-512.png

**Comment by janKir** [comment]
+1

**Comment by adrianogiannacco** [comment]
+1

**Comment by savadodesigns** [comment]
+1

**Comment by buzea** [comment]
+1

**Comment by miguelpeniche** [comment]
+1

**Comment by Amine-Smahi** [comment]
+1

**Comment by saskiab06** [comment]
+1

**Comment by Nicoolandgood** [comment]
+1

**Comment by lucajung** [comment]
+1

**Comment by raphaelfrota** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by KevinAshbyHN** [comment]
+1 for table icon

**Comment by giufiorenzano** [comment]
+1

**Comment by sensibleworld** [comment]
We've added these icons to 7.2:

https://fontawesome.com/icons/table-bar?f=classic&s=solid
https://fontawesome.com/icons/table-dining?f=classic&s=solid
